### PR TITLE
Fix broken `make images` process

### PR DESCRIPTION
**Pull request by eliheuer**
This fixes issue: https://github.com/googlefonts/googlefonts-project-template/issues/123

It also changes the `make images` workflow to reflect how people would want to use that feature. You don't want the images to be committed by Git every time you rebuild them for example.

The example DrawBot script and image have also been updated.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,11 +36,10 @@ test: venv build.stamp
 proof: venv build.stamp
 	. venv/bin/activate; mkdir -p out/ out/proof; diffenator2 proof $(shell find fonts/ttf -type f) -o out/proof
 
-images: venv build.stamp $(DRAWBOT_OUTPUT)
-	git add documentation/*.png && git commit -m "Rebuild images" documentation/*.png
+images: venv $(DRAWBOT_OUTPUT)
```
Comment (simoncozens) [inline Makefile:39]: These lines were fine, weren't they? You have to ensure the fonts are built before running the drawbot scripts, and it's good to check them in afterwards.

Comment (eliheuer) [reply Makefile:39]: There is already the `make build` command, in my experience combining these two commands causes more problems than it solves, and I almost always disable the pervious configuration. See [https://github.com/eliheuer/gf-specimen-images](https://github.com/eliheuer/gf-specimen-images) as an [example](https://github.com/eliheuer/gf-specimen-images/blob/7e861960433d09d141fcc5962e06a3553e10487f/Makefile#L39).

Anyone familiar enough with  the GFPT will know how to run `make build` before they start using `make images`. Sometime you will want to make images based on the ttfs in `fonts` directory and you might not want to rebuild because you are generating images to use in visual diffs or you want to see what the fonts look like in their current state.

If the source file is broken, the images will not build currently. 

Basically this is combining two actions that you do not always want to be combined.
